book/languages/js/intro.md

```diff
@@ -0,0 +1,44 @@
+# Introduction
+
+## Inception
+
+**JavaScript (JS)**, but the official name is **ECMAScript**. <br> This language is really fast scripting language using specially for web pages, for **frontend** (Vue.js, React.js etc.) and for **backend** (node.js etc.). It can be used also for creating desktop and mobile native apps (Proton, Electron).
+**Brednan Eich** created that language, and it firstly appeared in **December 4, 1995**. JS became an **ECMA standard in 1997**. <br> File extension is **.js**
+
```

Comment:
Inceptio is so fine.

Reply:
Removing inline html <br> for resolving travis error